### ISSUE TITLE
Incorrect heading structure in the About Publication Panel

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/readium-desktop#3353]
Reported by a user:

> The "About publication" menu has unconventional heading structure, as there are skipped heading levels (level 2 to level 4). This can cause screen reader users because the structure leads them to believe that content is missing or that they've jumped past sections.

<img width="414" height="191" alt="Image" src="https://github.com/user-attachments/assets/a502fc49-fa5f-49c2-89c2-306aa33d9d52" />
